nibabel/analyze.py

```diff
@@ -402,7 +402,7 @@ def from_header(klass, header=None, check=True):
                                      klass))
         obj.set_data_dtype(header.get_data_dtype())
         obj.set_data_shape(header.get_data_shape())
-        obj.set_zooms(header.get_zooms())
+        obj.set_zooms(header.get_zooms(units='raw'), units='raw')
```

Comment:
Perhaps using `units='norm'` in both cases would be better, as it would also entail setting sensible units for NIfTI. But that may be making a bad assumption if you're in a non-temporal 4th dimension case (ppm, rads, hz).